sdk/webpubsub/azure-messaging-webpubsub/src/test/java/com/azure/messaging/webpubsub/WebPubSubServiceClientTests.java

```diff
@@ -0,0 +1,184 @@
+    }
+
+    @Test
+    public void testSendToUserString() {
+        assertResponse(client.sendToUserWithResponse("test_user", "Hello World!", WebPubSubContentType.TEXT_PLAIN, Context.NONE), 202);
+    }
+
+    @Test
+    public void testSendToUserBytes() {
+        assertResponse(client.sendToUserWithResponse("test_user", "Hello World!".getBytes(), WebPubSubContentType.TEXT_PLAIN, Context.NONE), 202);
+    }
+
+    @Test
+    public void testSendToConnectionString() {
+        assertResponse(client.sendToConnectionWithResponse("test_connection", "Hello World!", WebPubSubContentType.TEXT_PLAIN, Context.NONE), 202);
+    }
+
+    @Test
+    public void testSendToConnectionBytes() {
+        assertResponse(client.sendToConnectionWithResponse("test_connection", "Hello World!".getBytes(), WebPubSubContentType.TEXT_PLAIN, Context.NONE), 202);
+    }
+
+    @Test
+    public void testSendToConnectionJson() {
+        assertResponse(client.sendToConnectionWithResponse("test_connection", "{\"data\": true}", null, Context.NONE),
+            202);
+    }
+
+    @Test
+    public void testSendToAllJson() {
+        assertResponse(client.sendToAllWithResponse("{\"boolvalue\": true}", null, null, Context.NONE), 202);
+        assertResponse(client.sendToAllWithResponse("{\"stringvalue\": \"testingwebpubsub\"}", null, null,
+            Context.NONE), 202);
+        assertResponse(client.sendToAllWithResponse("{\"intvalue\": 25}", null, null, Context.NONE), 202);
+        assertResponse(client.sendToAllWithResponse("{\"floatvalue\": 55.4}", null, null, Context.NONE), 202);
+    }
+
+    @Test
+    public void testRemoveNonExistentUserFromHub() {
+        // TODO (jogiles) can we determine if this user exists anywhere in the current hub?
+        Response<Void> removeUserResponse =
+            client.removeUserFromAllGroupsWithResponse("testRemoveNonExistentUserFromHub", Context.NONE);
+        assertEquals(200, removeUserResponse.getStatusCode());
+    }
+
+    /*****************************************************************************************************************
+     * Sync Tests - WebPubSubGroup
+     ****************************************************************************************************************/
+
+    @Disabled
+    @Test
+    public void testRemoveNonExistentUserFromGroup() {
+        WebPubSubGroup javaGroup = client.getGroup("java");
+
+        // assertFalse(javaGroup.checkUserExists("testRemoveNonExistentUserFromGroup"));
+        Response<Void> removeUserResponse = javaGroup.removeUserWithResponse("testRemoveNonExistentUserFromGroup", Context.NONE);
+        assertEquals(200, removeUserResponse.getStatusCode());
+        // assertFalse(javaGroup.checkUserExists("testRemoveNonExistentUserFromGroup"));
+    }
+
+    @Disabled
+    @Test
+    public void testAddAndRemoveUserToGroup() {
+//        WebPubSubServiceClient simpleChat = client.getHubClient("test");
+        WebPubSubGroup javaGroup = client.getGroup("java");
```

Comment:
Looks like this test is not completed yet. Also found codes like `SignalRHubAsyncClient` which are supposed to clean up?

Reply:
This test is now removed as this requires setting up a websocket client which is out of scope for playback tests.